.github/workflows/tool-tests.yml

```diff
@@ -1,38 +1,23 @@
-# This is a basic workflow to help you get started with Actions
+name: Tool Tests
 
-name: CI
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
 
-# Controls when the action will run. 
 on:
-  # Triggers the workflow on push or pull request events but only for the develop branch
   push:
-    branches: [ develop ]
-  pull_request:
-    branches: [ develop ]
+    branches: "**"
 
-  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
```

Comment:
I think it makes sense to test that we can actually install all the tools here in a separate and parallel job. See https://github.com/citusdata/tools/commit/ecf852b498e26d3a62d0edd94cb5dd6b81ba7092 for an idea

Reply:
Each jon runs on different machine so that I think seperating the step will increase the job time since each job needs provisioning of a machine

Reply:
Let me clarify myself. I do not suggest that we should separate any of your `unit_test_execution` steps. I want to add a parallel job that will actually check that we can build and install all other tools in this repo.

```yml
  make-install:
    runs-on: ubuntu-latest

    steps:
      - name: Check out repository
        uses: actions/checkout@v2
      - name: Make and install all scripts
        run: make && sudo make install
```

Reply:
Fixed